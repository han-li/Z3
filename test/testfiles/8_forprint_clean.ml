for i=1 to 10 do  print_string "NUMBER "; print_int i; print_newline () done; print_newline() ;;
