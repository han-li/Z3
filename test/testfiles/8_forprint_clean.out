NUMBER 1
NUMBER 2
NUMBER 3
NUMBER 4
NUMBER 5
NUMBER 6
NUMBER 7
NUMBER 8
NUMBER 9
NUMBER 10

